### PR TITLE
Relevé pour un tiers

### PR DESCRIPTION
**Pull request by rldhont**
Ajout de la possibilité de faire un relevé pour un tiers.

Funded by Calvados https://www.calvados.fr

### CODE REVIEW

File: cadastre/install/install_1_6.php [hunk 1]
```diff
@@ -19,6 +19,7 @@ public function install()
 
             // Add right subject
             jAcl2DbManager::addSubject('cadastre.acces.donnees.proprio', 'cadastre~search.cadastre.acces.donnees.proprio', 'cadastre.subject.group');
+            jAcl2DbManager::addSubject('cadastre.acces.donnees.proprio.simple', 'cadastre~search.cadastre.acces.donnees.proprio.simple', 'cadastre.subject.group');
```
Comment (mdouchin) [inline cadastre/install/install_1_6.php:22]: Please add a new file upgrade-XXX.php which will add the needed rights for already installed cadastre module

File: cadastre/locales/en_US/search.UTF-8.properties [hunk 2]
```diff
@@ -25,4 +25,5 @@ dock.title=Cadastre
 
 cadastre.subject.group.name=Droits liés au cadastre
 cadastre.use.search.tool=Utiliser le panneau de recherche
-cadastre.acces.donnees.proprio=Accéder aux données de propriétaires
+cadastre.acces.donnees.proprio.simple=Accéder aux données de propriétaires simplifiés
+cadastre.acces.donnees.proprio=Accéder aux données de propriétaires avancées
```
Comment (mdouchin) [inline cadastre/locales/en_US/search.UTF-8.properties:29]:
```suggestion
cadastre.acces.donnees.proprio.simple=Accéder aux données de propriétaires simplifiées
cadastre.acces.donnees.proprio=Accéder aux données de propriétaires avancées
```

File: cadastre/locales/fr_FR/search.UTF-8.properties [hunk 3]
```diff
@@ -25,4 +25,5 @@ dock.title=Cadastre
 
 cadastre.subject.group.name=Droits liés au cadastre
 cadastre.use.search.tool=Utiliser le panneau de recherche
-cadastre.acces.donnees.proprio=Accéder aux données de propriétaires
+cadastre.acces.donnees.proprio.simple=Accéder aux données de propriétaires simplifiés
+cadastre.acces.donnees.proprio=Accéder aux données de propriétaires avancées
```
Comment (mdouchin) [inline cadastre/locales/fr_FR/search.UTF-8.properties:29]:
```suggestion
cadastre.acces.donnees.proprio.simple=Accéder aux données de propriétaires simplifiées
cadastre.acces.donnees.proprio=Accéder aux données de propriétaires avancées
```

**Review by mdouchin — CHANGES_REQUESTED** [review]

**Comment by rldhont** [comment]
@mdouchin thanks for the review

**Comment by rldhont** [comment]
@mdouchin changes requested done.

**Review by mdouchin — APPROVED** [review]

**Review by laurentj — APPROVED** [review]
ok for the upgrader

**Comment by rldhont** [comment]
Merci @laurentj

**Pull request merged**